test/known_issues/test-vm-inherited_properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+// Ref: https://github.com/nodejs/node/issues/5350
```

Comment:
We usually don't include `Ref:` when linking to issues. I can only find two tests that do that.  (For consistency they probably shouldn't)